openedx/core/djangoapps/catalog/management/commands/tests/test_sync_course_runs.py

```diff
@@ -38,18 +39,28 @@ def get_course_overview_marketing_url(self, course_id):
         """
         return CourseOverview.objects.get(id=course_id).marketing_url
 
+    def get_financial_aid_eligible_course_overviews(self, course_id):  # pylint: disable=invalid-name
+        """
+        Get course overview marketing url.
```

Comment:
Also, you don't need this method considering it isn't really needed by more than one test.

```python
course_overview = CourseOverview.objects.get(id=course_id)

# Do stuff that changes the model

course_overview.refresh_from_db()
course_overview.eligible_for_financial_aid
```